src/main/java/macrobase/analysis/classify/StaticThresholdClassifier.java

```diff
@@ -0,0 +1,31 @@
+package macrobase.analysis.classify;
+
+import macrobase.analysis.result.OutlierClassificationResult;
+import macrobase.conf.MacroBaseConf;
+import macrobase.conf.MacroBaseDefaults;
+import macrobase.datamodel.Datum;
+
+import java.util.Iterator;
+
+public class StaticThresholdClassifier extends OutlierClassifier {
+
+    private Double threshold;
+    private Double thresholdSquared;
+
+    public StaticThresholdClassifier(MacroBaseConf conf, Iterator<Datum> input) {
+        super(conf, input);
+        threshold = conf.getDouble(MacroBaseConf.OUTLIER_STATIC_THRESHOLD, MacroBaseDefaults.OUTLIER_STATIC_THRESHOLD);
+        thresholdSquared = threshold * threshold;
+    }
+
+    @Override
+    public boolean hasNext() {
+        return input.hasNext();
+    }
+
+    @Override
+    public OutlierClassificationResult next() {
+        Datum nextInput = input.next();
+        return new OutlierClassificationResult(nextInput, thresholdSquared > nextInput.getMetrics().getNorm());
```

Comment:
no real distinction, the only distinction is that this datum is a Datum that had gone through the feature transformation, so the values in the metrics are not the values that we got out of postgres, rather the values that we gave them based on transformations applied, e.g. windowed average, Zscore, inverse of the value, etc. Whatever we have defined in the pipeline.